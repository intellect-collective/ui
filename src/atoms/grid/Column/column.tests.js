import React from 'react';
import {
    Column
} from '../../../..';

describe('Column', () => {
    it('Can display', () => {
        const wrapper = mount(
            <Column sm-width="12" md-width="6" lg-width="4">test</Column>
        );
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('.col-sm-12').exists()).toBe(true);
        expect(wrapper.find('.col-md-6').exists()).toBe(true);
        expect(wrapper.find('.col-lg-4').exists()).toBe(true);
    });
    it('Can use default medium width', () => {
        const wrapper = mount(<Column width="8" />);
        expect(wrapper.find('.col-md-8').exists()).toBe(true);
    });
    it('Can change classname', () => {
        const wrapper = mount(<Column className="my-column" />);
        expect(wrapper.find('.my-column').exists()).toBe(true);
    });
});
