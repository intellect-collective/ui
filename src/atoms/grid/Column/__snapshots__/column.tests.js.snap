// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column Can display 1`] = `
<Column
  lg-width="4"
  md-width="6"
  sm-width="12"
>
  <div
    className="col-sm-12 col-md-6 col-lg-4"
  >
    test
  </div>
</Column>
`;
